packages/lib/src/components/UIElement.tsx

```diff
@@ -45,6 +48,25 @@ export class UIElement<P extends UIElementProps = any> extends BaseElement<P> im
         return state;
     }
 
+    /* eslint-disable-next-line */
+    protected submitAnalytics(obj = null) {
```

Comment:
The `obj` parameter isn't not used in this function

Reply:
But it will be in the function that `ThreeDS2DeviceFingerprint` & `ThreeDS2DeviceChallenge` override